content/management/configuration-api/v3.2/index.mdx

```diff
@@ -2785,6 +2785,230 @@ curl -X POST \
 
 </Section>
 
+# Auto access rules
+
+Auto access rules allow to assign groups to customer session according to their tracking url and geolocation.  Those rules are
+checked on **Login** and **Update Customer Page** and resulting groups stored in customer session.  When that session starts chat with **Start Chat**
+or **Activate Chat** without `chat.access` stored session groups are used as new thread's access.
+
+## Methods
+
+<Section>
+
+<Text>
+
+### Add auto access rules
+
+
+#### Specifics
+
+|                |                                                                               |
+| -------------- | ----------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.2/configuration/action/add_auto_access_rules` |
+| Required scopes| `access_rules:rw`                                                             |
+
+
+| Parameter       | Required     | Data type | Notes                         |
+| --------------- | ------------ | --------- | ----------------------------- |
+| `access`        | Yes          | `object`  | Destination access            |
```

Comment:
not sure if `destination access` says clearly enough `who will have the access`

Reply:
I guess it should be clear from the description on top, where it was decided to not mention customer access at all and concentrate on created thread if needed.